lighthouse-core/audits/byte-efficiency/uses-responsive-images.js

```diff
@@ -41,9 +41,9 @@ class UsesResponsiveImages extends ByteEfficiencyAudit {
       description: 'Properly size images',
       informative: true,
       helpText:
-        'Image sizes served should be based on the device display size to save network bytes. ' +
-        'Learn more about [responsive images](https://developers.google.com/web/fundamentals/design-and-ui/media/images) ' +
-        'and [client hints](https://developers.google.com/web/updates/2015/09/automating-resource-selection-with-client-hints).',
+        'Serve images that are smaller than the user\'s viewport to save cellular data ' +
```

Comment:
haha, well I said nitpicky because the linked doc describes it correctly with "Lighthouse compares the size of the rendered image against the size of the actual image." and the previous description wasn't exactly illuminating here and just said "based on the display size" which could be interpreted as the "screen size" instead of "rendered size"

but yeah I'm glad we're fixing it :)